core/test/utils/index.js

```diff
@@ -251,12 +251,20 @@ fixtures = {
     // Creates a client, and access and refresh tokens for user 3 (author)
     createTokensForUser: function createTokensForUser() {
         return db.knex('clients').insert(DataGenerator.forKnex.clients).then(function () {
-            return db.knex('accesstokens').insert(DataGenerator.forKnex.createToken({user_id: 3}));
+            return db.knex('accesstokens').insert(DataGenerator.forKnex.createToken({id:3, user_id: 3}));
```

Comment:
Yes, it is. But I let the code like that for now to have one thing less to check. I'll change it when the test will pass.
I even found a couple of errors in models/base/token where the promises are bad returned but I'll open a new PR for this